### ISSUE TITLE
Icon Request: fa-fediverse

### ISSUE THREAD

**Issue by marcelcosta** [FortAwesome/Font-Awesome#16252]
Fediverse represents the social network applications that use ActivityPub protocol (such as Mastodon) that allows to communicate between them.

The icon would be use to refer to an account from a social application compatible with that protocol.
[Fediverse_logo_proposal.pdf](https://github.com/FortAwesome/Font-Awesome/files/4267952/Fediverse_logo_proposal.pdf)

**Comment by wznmickey** [comment]
+1

**Comment by ThatOneCalculator** [comment]
+1 on this. It would also be nice to have other Fediverse services as icons (Misskey, OwnCast, PeerTube, PixelFed, Pleroma, etc), but at the *very* least a Fediverse icon should be added to represent the rest of the Fediverse (which can help clear up the misconception that the Fediverse is only Mastodon, which it isn't.)

**Comment by pmbrent** [comment]
Wanted to ➕ this as instability at Twitter is making Fediverse/Mastodon a more common link-out destination for people's social media link sections. I use Font-Awesome icons for my others and would be happy to seamlessly include this one.

**Comment by wznmickey** [comment]
自动回复：已收到，谢谢。

**Comment by idnorton** [comment]
A Mastodon icon was added some time ago - details in https://github.com/FortAwesome/Font-Awesome/issues/10897

**Comment by alexlancaster** [comment]
> A Mastodon icon was added some time ago - details in #10897

this is true, but Fediverse =/= Mastodon. I think the OP was suggesting adding an icon for a generic Fediverse link

**Comment by yannicka** [comment]
With the increasing importance of ActivityPub, it would be nice to have a dedicated icon for the Fediverse/ActivityPub.

**Comment by ThatOneCalculator** [comment]
Also to clarify, there's more than just ActivityPub on the Fediverse 😌

**Comment by gaycookie** [comment]
I would love to bump this request, especially because I think its important to understand that Mastodon is not the only software that runs the Fediverse and that even the Fediverse has it's own "icon".
As what @ThatOneCalculator @alexlancaster @marcelcosta mentioned!

I wish I knew my way around with SVGs or else I would've actually made SVGs and submit them.

**Comment by Neustradamus** [comment]
Any progress on it?

**Comment by trickstar13** [comment]
➕ 1
I need this icon to express Fediverse, not only the Mastodon.

**Comment by Neustradamus** [comment]
At all, there is ActivityPub too:
- https://github.com/FortAwesome/Font-Awesome/issues/19668

Please vote too.

**Comment by alphatownsman** [comment]
it would be really nice to have this, as Fediverse is Mastodon + Threads + dozens of other softwares running on a global network.

**Comment by jernst** [comment]
I would use it.

**Comment by shleeable** [comment]
+1 Yes please.

**Comment by Hamish-McLean** [comment]
This would be great!

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.

**Comment by sebix** [comment]
Started a discussion there: https://github.com/FortAwesome/Font-Awesome/discussions/21089

**Comment by Neustradamus** [comment]
@sebix: Thanks :)
